server/server.js

```diff
@@ -3,7 +3,9 @@ const app = express();
 const port = 3003;
```

Comment:
@chanceknguyen should the port be written as process.env.PORT || 3003 or would that be something to consider later for deployment?

Reply:
@owenwexler You know, I actually have never utilized that. After doing research, it seems like that is the convention that would allow for the deployment app to pass in a port, otherwise the default would be 3003. Thanks for the insight! I will merge the pull requests and then make that change.